### PR TITLE
Hardwarehelper designed using Bootstrap

### PR DESCRIPTION
**Pull request by simjeeh**
Design the basics for the website. We will need to do more work on it once we figure out the store.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,322 @@
+<!DOCTYPE html>
```
Comment (zsimjee) [inline index.html:1]: why was this necessary to move to a diff file? Instead, why not have an index.html with a nav, and the cabinets.html should be one of the items in the nav

File: js/init.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+(function($){
```
Comment (zsimjee) [inline js/init.js]: why...

also, if we do have a js bucket, we should refactor the knapsack formula into it from cabinets.html

File: js/init.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+  $(function(){
+
+    $('.button-collapse').sideNav();
```
Comment (zsimjee) [inline js/init.js]: we shouldn't need this

File: js/init.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+    $('.parallax').parallax();
```
Comment (zsimjee) [inline js/init.js]: I don't like parallax

File: js/materialize.js [hunk 5]
```diff
@@ -0,0 +1,10021 @@
+/*!
+ * Materialize v0.100.2 (http://materializecss.com)
+ * Copyright 2014-2017 Materialize
+ * MIT License (https://raw.githubusercontent.com/Dogfalo/materialize/master/LICENSE)
+ */
+var _createClass = function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; }();
```
Comment (zsimjee) [inline js/materialize.js]: Why would you make the choice to import this file vs link to it via CDN? Why is this a necessary dependency in the first place?

File: js/materialize.min.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
...
...verlay[0].parentNode&&this.$overlay[0].parentNode.removeChild(this.$overlay[0]),this.$el[0].M_Modal=void 0,n._count--}},{key:"setupEventHandlers",value:function(){this.handleOverlayClickBound=this.handleOverlayClick.bind(this),this.handleModalCloseClickBound=this.handleModalCloseClick.bind(this),1===n._count&&document.body.addEventListener("click",this.handleTriggerClick),this.$overlay[0].addEventListener("click",this.handleOverlayClickBound),this.$el[0].addEventListener("click",this.handleModalCloseClickBound)}},{key:"removeEventHandlers",value:function(){0===n._count&&document.body.removeEventListener("click",this.handleTriggerClick),this.$overlay[0].removeEventListener("click",this.handleOverlayClickBound),this.$el[0].removeEventListener("click",this.handleModalCloseClickBound)}},{key:"handleTriggerClick",value:function(e){var i=t(e.target).closest(".modal-trigger");if(e.target&&i.length){var n=i[0].getAttribute("href");n=n?n.slice(1):i[0].getAttribute("data-target");var o=document.getElementById(n).M_Modal;o&&o.open(i),e.preventDefault()}}},{key:"handleOverlayClick",value:function(){this.options.dismissible&&this.close()}},{key:"handleModalCloseClick",value:function(e){var i=t(
```
Comment (zsimjee) [inline js/materialize.min.js]: same as above

**Review by zsimjee — CHANGES_REQUESTED** [review]
Yeah please clean this up. I don't know why we're using materialize... don't really see what it buys us. I think we can do this using just jquery+bootstrap.

Also, collapse similar commits. You've posted a review request for 27 commits... I don't see why...

**Comment by simjeeh** [comment]
Merged commits together and have the basic website designed. Waiting for the backend to implement the store.

**Comment by simjeeh** [comment]
added a loader as well and made UI changes

**Review by zsimjee — APPROVED** [review]

**Review by zsimjee — APPROVED** [review]

**Pull request merged**